lua/rust-tools/fly_check.lua

```diff
@@ -0,0 +1,8 @@
+local M = {}
+
+function M.fly_check()
+  local params = vim.lsp.util.make_text_document_params()
+  vim.lsp.buf_notify(0, "rust-analyzer/runFlycheck", params)
```

Comment:
```suggestion
  vim.lsp.buf_notify(0, "rust-analyzer/runFlycheck", {
    textDocument = vim.lsp.util.make_text_document_params()
  })
```

Ref: https://github.com/rust-lang/rust-analyzer/blob/master/docs/dev/lsp-extensions.md#controlling-flycheck